test/compile/data/bin.test.ts

```diff
@@ -102,4 +103,64 @@ describe('compile/data/bin', function() {
       expr: 'format(datum["bin_maxbins_10_Rotten_Tomatoes_Rating_start"], \'s\') + \' - \' + format(datum["bin_maxbins_10_Rotten_Tomatoes_Rating_end"], \'s\')'
     });
   });
+
+  it ('should add bin transform and correctly apply bin for binned field without custom extent', () => {
```

Comment:
Are you just adding this to improve test coverage? 
(It seems irrelevant to what you are adding in this PR)